starter-template/README.md

```diff
@@ -10,97 +10,60 @@ You can then go ahead and edit the template to fit your needs.
 
-- `dist` - Your Dapp front-end files should be placed under this folder. You can see there we already creaated a basic HTML page imorting JQuery and the compiled js file from `src` (named `main.js`).
+## Running your project on local testnet:
 
-## Running your project:
+After downloading the project and docker:
 
-Enter the project folder from the terminal and type the following:
-
-After downloading the project:
+- copy the `example.env` from `data` directory as `.env` in the project directory
+- Update the `.env` file with your `SEED_PHRASE` and <TESTNET-NAME>`_INFURA_URL`
+- Enter the project folder from the terminal (starter-template) and type the following:
 
 ```
-npm install
-npm install -g nps
-npm install -g npx
+npm run launch:docker
+npm run logs:graph
 ```
 
-### Run on a local testnet:
-
-```
-npm run ganache
-```
+This would build the docker images and start the docker container for `ganache`, `starter-app` with webpack server, `graph-node` for indexing blockchain events, `ipfs` where subgraph mappings reside and `postgres` database where blockchain events are stored as described by the schema
 
-Then on a different terminal window (but still in your project folder) you can choose one of the 2:
-
-1. Run all with one command:
+Wait for the logs screen to say *Starting JSON-RPC admin server at: ....*, then you can either exit the logs or in a different terminal window type
 
 ```
-npm run launch-local
+npm run deploy-graph
 ```
 
-Then open your web browser and type `http://localhost:3000/`
+This will deploy the subgraph and if you are still following the graph-node logs you can see it syncing the blockchain events
 
-2. Run the commands one by one as follows:
+Open your web browser and type *http://localhost:3000/* This is your react Dapp
 
-```
-npm run migrate-daostack
-rm -rf build
-npm run truffle-migrate
-```
-
-### Deploy and use on Kovan network:
+In another browser window/tab type *http://localhost:8000/subgraphs/name/daostack* This is an interface to the subgraph and you can type graphQL queries here to fetch data from the `postgres` database
 
-1. Enter the `truffle.js` file and make the changes as instructed there.
-2. Open `src/index.js` and uncomment the lines to configure Kovan (as instructed there).
-3. Open terminal at the project folder
-4. Run the following commands:
+To learn more about subgraphs check out: [DAOstack Subgraph](https://github.com/daostack/subgraph), [TheGraph Project](https://thegraph.com/docs/quick-start)
 
-```
-rm -rf build
-truffle migrate --network kovan-infura
-```
+### Deploy and use on test network:
 
-5. Copy the Avatar and Voting Machine addresses from the end of the `truffle migrate` logs.
-6. Open `src/index.js` and pasted the copied addresses in the `avatarAddress` and `votingMachineAddress`.
-7. On the terminal window, run the following command:
+- Update the `.env` file with your `SEED_PHRASE` and <TESTNET-NAME>`_INFURA_URL`
```

Comment:
<TESTNET-NAME>`_INFURA_URL` -> `<TESTNET-NAME>_INFURA_URL`
(just to make it look nicer in markdown)